.github/workflows/ci-build.yml

```diff
@@ -39,7 +39,6 @@ jobs:
       fail-fast: true
       matrix:
         ember-try-scenario:
-          - ember-lts-3.8
```

Comment:
Why do we need to drop this?

Reply:
Again, as I mentioned to you via chat, this scenario was hanging, and rather than my worrying about fixing it (since we're removing support for this in another PR), I'd rather just remove it.

Reply:
Gotcha. Yeah, its fairly bad but since this PR itself is definitely not breaking anything I think that is fine.